api/FetchEvent.json

```diff
@@ -0,0 +1,439 @@
+{
+  "api": {
+    "FetchEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent",
+        "support": {
+          "webview_android": {
+            "version_added": "40"
+          },
+          "chrome": {
+            "version_added": "40"
+          },
+          "chrome_android": {
+            "version_added": "40"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "44",
+            "notes": "Service workers (and <a href='https://developer.mozilla.org/docs/Web/API/Push_API'>Push</a>) have been disabled in the <a href='https://www.mozilla.org/en-US/firefox/organizations/'>Firefox 45 and 52 Extended Support Releases</a> (ESR.)"
+          },
+          "firefox_android": {
+            "version_added": "44"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "27"
+          },
+          "opera_android": {
+            "version_added": "27"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "preloadResponse": {
+        "__compat": {
+          "description": "<code>preloadResponse</code> property",
+          "support": {
+            "webview_android": {
+              "version_added": "59"
+            },
+            "chrome": {
+              "version_added": "59"
+            },
+            "chrome_android": {
+              "version_added": "59"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "46"
+            },
+            "opera_android": {
+              "version_added": "46"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "client": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FetchEvent/client",
+          "support": {
+            "webview_android": {
+              "version_added": null
```

Comment:
Hold off on merging for now. I don't think this is null. Let me see if I can get an answer.

Reply:
I forgot this was a deprecated feature. So, never mind.

Reply:
👍